submissions/Nik3264/js-dom/style.css

```diff
@@ -0,0 +1,255 @@
+    clip-path: polygon(100% 0, 100% 100%, 0 100%, 0 100%);
+  }
+  100% {
+    transform: translateY(0);
+    opacity: 1;
+    -webkit-clip-path: polygon(100% 0, 100% 100%, 0 100%, 0 0);
+    clip-path: polygon(100% 0, 100% 100%, 0 100%, 0 0);
+  }
+}
+
+.header {
+  width: 100%;
+  height: 12vh;
+  padding-top: 1vh;
+  background-color: var(--light-brown);
+  color: var(--light-gray);
+  text-align: center;
+  text-transform: uppercase;
+  text-shadow: var(--grey-blue) 2px 5px 3px;
+  font-size: clamp(12px, 2vh, 34px);
+}
+
+.wrap {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: flex-start;
+  width: 100%;
+  height: 88vh;
+  background-color: var(--light-gray);
+  position: relative;
+  overflow-x: hidden;
+}
+
+.menu {
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  width: 30%;
+  height: 100%;
+  background-color: var(--light-gray);
+  cursor: pointer;
+  padding: 1.2em;
+  position: relative;
+  transition: all ease 0.2s;
+}
+
+.menu__item:hover {
+  color: var(--grey-blue);
+  font-size: 1.8em;
+  font-weight: 700;
+  transition: all ease 0.1s;
+}
+
+.menu__item {
+  background-color: var(--light-gray);
+  color: var(--dark-brown);
+  font-size: 1.7em;
+  margin-bottom: 15px;
+  height: 60px;
+  padding: 5px;
+  box-shadow: 0 4px 12px var(--dark-brown), 0 16px 20px var(--dark-brown);
+}
+
+.menu__item__active {
+  font-size: 1.8em;
+  font-weight: 700;
+}
+
+.content {
+  width: 70%;
+  height: 100%;
+  background-color: var(--light-gray);
+  position: relative;
+}
+
+.group-info {
+  height: 88vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: stretch;
+}
+
+.group-info__src {
+  flex-grow: 2;
+  align-self: center;
+  object-fit: cover;
+  max-width: 60%;
+  max-height: 50%;
+  margin-left: 20%;
+  margin-right: 20%;
+  border-radius: 2%;
+  box-shadow: 5px 15px 10px 3px;
+  animation: fade 0.7s;
+}
+
+.group-info__text,
+.group-info__header {
+  margin: 0 auto;
+  width: 80%;
+  text-align: center;
+  margin-top: 25px;
+  color: var(--dark-brown);
+}
+
+.group-info__header {
+  flex-grow: 0.1;
+  font-size: 2.2em;
+  text-transform: uppercase;
+  margin-bottom: 25px;
+  text-shadow: 2px 2px 2px var(--grey-blue);
+  animation: fade-text 0.5s ease-out;
+}
+
+.group-info__text {
+  flex-grow: 2;
+  font-size: 1.5em;
+  animation: fade-text 0.85s ease-out;
+}
+
+.menu__burger:before,
+.menu__burger:after,
+.menu__burger {
+  display: none;
+}
+
+.hidden {
+  display: none;
+  z-index: -10;
+}
+
+@media screen and (min-width: 1025px) {
+  .wrap {
+    font-size: 18px;
+  }
+}
+
+@media screen and (min-width: 769px) and (max-width: 1024px) {
+  .header {
+    font-size: 18px;
+  }
+
+  .wrap {
+    font-size: 14px;
+  }
+}
+
+@media screen and (min-width: 481px) and (max-width: 768px) {
+  .header {
+    height: 10vh;
+    font-size: 14px;
+  }
+
+  .wrap {
+    height: 90vh;
+    font-size: 11px;
+  }
+}
+
+@media screen and (max-width: 480px) {
```

Comment:
Mobile version should start from `768px`

Reply:
done